libraries/AP_Declination/tables.cpp

```diff
@@ -9,69 +9,69 @@ const float AP_Declination::SAMPLING_MAX_LAT = 90;
-    {6.42021f,8.49313f,9.96485f,11.21264f,12.15378f,12.34411f,11.39654f,9.00192f,4.80210f,-1.14083f,-7.63429f,-12.77860f,-15.31639f,-15.15258f,-12.98558f,-9.72317f,-6.02652f,-2.46224f,0.32036f,2.16718f,3.52576f,4.45316f,4.47022f,3.64413f,2.71916f,2.05267f,1.37415f,0.37187f,-1.18524f,-3.37771f,-5.55055f,-6.50029f,-5.64204f,-3.28034f,-0.00971f,3.47278f,6.42021f},
-    {4.55870f,7.84457f,10.59505f,12.78315f,14.21311f,14.53879f,13.38981f,10.37263f,5.13228f,-2.00167f,-9.27410f,-14.41195f,-16.39580f,-15.63899f,-13.13217f,-9.75841f,-6.05603f,-2.45211f,0.55836f,2.75052f,4.36042f,5.58048f,6.24404f,6.24213f,5.76940f,4.95204f,3.62521f,1.54168f,-1.40447f,-4.90584f,-7.98277f,-9.46456f,-8.87577f,-6.53558f,-3.08458f,0.80580f,4.55870f},
-    {3.13967f,7.31097f,11.07216f,14.15725f,16.20221f,16.79070f,15.47250f,11.72257f,5.14656f,-3.57391f,-11.94254f,-17.34882f,-19.11810f,-18.05435f,-15.26042f,-11.58179f,-7.54393f,-3.53438f,0.07849f,3.08157f,5.54519f,7.63184f,9.31427f,10.36791f,10.53101f,9.56965f,7.27456f,3.54700f,-1.35789f,-6.53724f,-10.58593f,-12.40763f,-11.80293f,-9.26734f,-5.52522f,-1.23338f,3.13967f},
-    {2.40982f,7.18541f,11.61646f,15.39834f,18.09395f,19.11444f,17.67695f,12.80844f,3.91551f,-7.49296f,-17.41503f,-23.01926f,-24.41774f,-22.89374f,-19.60750f,-15.34185f,-10.59502f,-5.72094f,-1.00157f,3.37937f,7.37061f,10.97982f,14.11553f,16.47981f,17.57833f,16.80075f,13.55567f,7.60935f,-0.25054f,-7.92815f,-13.21489f,-15.22877f,-14.33921f,-11.39247f,-7.22465f,-2.48217f,2.40982f},
-    {1.84909f,7.14349f,12.09954f,16.39700f,19.54576f,20.73345f,18.58921f,11.09809f,-2.76476f,-18.58691f,-29.30539f,-33.52891f,-33.25409f,-30.30365f,-25.79412f,-20.37504f,-14.44263f,-8.26365f,-2.03561f,4.09039f,9.99389f,15.55055f,20.57404f,24.74657f,27.54152f,28.12085f,25.24078f,17.56424f,5.48335f,-6.76322f,-14.61951f,-17.38523f,-16.44524f,-13.21307f,-8.68808f,-3.52579f,1.84909f},
-    {-0.07018f,5.11056f,9.81033f,13.43064f,14.95811f,12.44881f,2.42652f,-17.21607f,-37.22275f,-47.59912f,-50.02338f,-48.04885f,-43.68750f,-37.95581f,-31.39385f,-24.31250f,-16.90710f,-9.31264f,-1.63265f,6.04381f,13.62973f,21.02738f,28.11104f,34.69910f,40.50309f,45.02417f,47.32932f,45.58173f,36.48238f,17.86736f,-1.80184f,-12.43534f,-15.24263f,-13.75101f,-10.05982f,-5.28238f,-0.07018f},
-    {-177.79784f,-167.79784f,-157.79784f,-147.79784f,-137.79784f,-127.79784f,-117.79784f,-107.79784f,-97.79784f,-87.79784f,-77.79784f,-67.79784f,-57.79784f,-47.79784f,-37.79784f,-27.79784f,-17.79784f,-7.79784f,2.20217f,12.20217f,22.20217f,32.20217f,42.20217f,52.20217f,62.20217f,72.20217f,82.20217f,92.20217f,102.20217f,112.20217f,122.20217f,132.20217f,142.20217f,152.20217f,162.20217f,172.20217f,-177.79784f}
+const float AP_Declination::declination_table[19][37] = {
```

Comment:
[19][37] can become a potential conflict with the header that is [LAT_TABLE_SIZE][LON_TABLE_SIZE]

Reply:
fixed, thanks